src/error_estimation/adjoint_refinement_estimator.C

```diff
@@ -529,18 +502,12 @@ void AdjointRefinementEstimator::estimate_error (const System & _system,
                 (std::abs(local_contribution));
 
             } // End loop over elements
-
         } // End if belong to QoI set
-
     } // End loop over QoIs
 
   for (std::size_t j=0; j != system.qoi.size(); j++)
-    {
-      if (_qoi_set.has_index(j))
-        {
-          delete coarse_adjoints[j];
-        }
-    }
+    if (_qoi_set.has_index(j))
+      delete coarse_adjoints[j];
```

Comment:
I'm still thrilled by the number of lines of code this PR strips, but slipping in unrelated whitespace and brace removal doesn't count for your final score.

Reply:
The brace removal counts! Since we don't have to have the extra line for dereferencing an iterator, it allowed several extra braces to be collapsed.

Reply:
Most of the brace removal counts, but this set was an exception.